docs/docs/page-query.md

```diff
@@ -2,7 +2,7 @@
 title: Querying data in pages with GraphQL
 ---
 
-Gatsby's `graphql` tag enables page components to retrieve data via GraphQL query.
+Gatsby's `graphql` tag enables page components to retrieve data via GraphQL queries.
```

Comment:
```suggestion
Gatsby's `graphql` tag enables page components to retrieve data via a GraphQL query.
```

We only support one per component so I guess this is better wording

Reply:
I agree 👍